### PR TITLE
Show all super type properties in first level types.

### PR DESCRIPTION
**Pull request by juancoen**

### CODE REVIEW

**Comment by juancoen** [comment]
screen shot 2016-04-01 at 5 50 30 pm
screen shot 2016-04-01 at 5 50 41 pm

File: src/common/inspector/types.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      }
+    } else {
+      return properties;
+    }
+  }
+
+  function mergeType(type, types) {
+    if (!isNativeType(type.type[0])) {
+      var resultingType = angular.copy(type);
+      resultingType.properties = getSuperTypesProperties(type.properties, type.type[0], types);
```
Comment (xaka) [inline src/common/inspector/types.js]: Actual type properties should come in last? If C inherits from A and B, then the final set of properties should be merge of {} + A + B + C, right?

Comment (juancoen) [inline src/common/inspector/types.js]: yes, the order of the properties is done inside `getSuperTypesProperties` :
`properties = Object.assign({}, superType.properties, properties);`

That way, super type properties always come first and property overrides in child types work

**Comment by xaka** [comment]
LGTM :+1:

**Pull request merged**